core/object/object.cpp

```diff
@@ -747,6 +759,14 @@ Variant Object::callv(const StringName &p_method, const Array &p_args) {
 	if (ce.error != Callable::CallError::CALL_OK) {
 		ERR_FAIL_V_MSG(Variant(), "Error calling method from 'callv': " + Variant::get_call_error_text(this, p_method, argptrs, p_args.size(), ce) + ".");
 	}
+
+	// Destroy the copies of the arguments if any
+	if (args) {
```

Comment:
Needs to be moved up above the error check or you'll have leaks

Reply:
Perhaps I should use another macro then. If I destroy the arguments before the, the `argptrs` will be pointing to pointers of destroyed Variants.

Reply:
That would be necessary yes, you'd have to make it `ERR_PRINT` and then have the return under